docs/docs/develop/glob-patterns.md

```diff
@@ -72,21 +84,17 @@ When ingesting data that matches a glob pattern, Rill processes the data in batc
 
 1. **Adding new columns**: If the previously ingested data has fewer columns than the new files, Rill adds the new columns to the schema.
 2. **Automatic datatype conversion**: Rill automatically converts the datatype of a column to match the datatype in all files. This ensures consistency across the ingested data.
-3. **Handling missing columns**: If columns were present in the previously ingested data but are not present in the new data, Rill adds null values for those columns.
 
-By default, these schema relaxation features are enabled in Rill. However, if you prefer to disable them, you can set the following properties to false in the source.yaml file:
+    NOTE: The datatype is only relaxed to wider types like Int32 ==> Int64 or double ==> varchar and would never be a breaking change (eg: varchar to int or double to int)
```

Comment:
1. Keep this as one line
2. Check punctuation
3. Use `code` tags and correct SQL type names for the types listed